packages/template-retail-react-app/app/pages/account/wishlist/partials/wishlist-primary-action.jsx

```diff
@@ -63,7 +63,10 @@ const WishlistPrimaryAction = () => {
             {isMasterProduct ? (
                 <>
                     <Button w={'full'} variant={'solid'} onClick={onOpen}>
-                        Select Options
+                        <FormattedMessage
+                            defaultMessage="Select Options"
+                            id="wishlist_primary_action.button.select_options"
```

Comment:
Great message id here btw. I don't think we've documented the id naming convention in any of the READMEs.

But the gist of it is typically a message id has the following items:
1. name of the component or page
2. UI element type ← there's actually a list of expected UI element types in some internal document (I can find it if you're curious)
3. a summary or a short phrase of the message